deploy/helm/demarkus-server/README.md

```
@@ -0,0 +1,144 @@
+helm install team-a ./deploy/helm/demarkus-server \
+  --set storage.className=premium-rwo \
+  --set storage.size=20Gi \
+  --set tls.certManager.enabled=true \
+  --set 'tls.certManager.dnsNames[0]=team-a.library.example.com' \
+  --set 'tls.certManager.issuerRef.name=letsencrypt-prod' \
+  --set 'service.annotations.cloud\.google\.com/neg=\{"exposed_ports":\{"6309":\{\}\}\}'
+```
+
+## Values
+
+### Image
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)